addon/utils.js

```diff
@@ -150,3 +150,73 @@ export function recursiveObjectCreate (object) {
 
   return object
 }
+
+/**
+ * mine an object for a value
+ * @param {Object} obj the object to mine
+ * @param {String} valuePath the path to the value in the object
+ * @returns {String} the value
+ */
+export function findValue (obj, valuePath, startPath = '') {
+  const depths = startPath.split('.')
+  const valueLevels = valuePath.split('./')
+  const parentLevels = valueLevels.filter((element) => {
+    return element === '.'
+  })
+  const valueKey = valueLevels.pop()
+  const absValuePath = _.without(depths.slice(0, depths.length - parentLevels.length - 1), '', '.').join('.')
+  const absValueKey = [absValuePath, valueKey].join('.')
+  return _.get(obj, absValueKey)
+}
+
+/**
+ * populates values in an Orchestrate-style 'q' or 'p' queryParam valuePath
+ * @param {Object} valueObj the value object to mine for query values
+ * @param {String} queryValue the queryParam value to parse
+ * @returns {String} the populate queryValue
+ */
+export function parseOrchFilters (valueObj, queryValue, startPath = '') {
```

Comment:
I'm surprised to see this logic here. I thought that schema authors would know whether they're talking to an orch style API or not and should just fill out the queries properly. Does it make sense for bunsen to be orch API aware? How does it decide to use this or not? I guess I'll keep looking to see where it's called.